app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is currently a class attribute. It should be an instance attribute to ensure it is managed correctly for each `Animal` instance. Consider moving it to the `__init__` method.